src/server/models/list.ts

```diff
@@ -12,9 +15,12 @@ import {
 } from "./types";
 
 export async function findUserLists(
-  email: string
+  userData: User
```

Comment:
FWIW - not necessary. req.user is already `AuthenticatedUser`. This is reinitialising this class when it doesn't need to be. 

The easiest thing to do is just add this to function as a method to the `AuthenticatedUser` class.